test/test_scanner.py

```diff
@@ -666,6 +662,12 @@ def test_binaries(self, binary, package, version, are_in, not_in):
                         "gnutls-cli",
                         "2.3.11",
                     ),
+                    (
+                        "https://kojipkgs.fedoraproject.org/packages/openssh/7.7p1/1.fc29/x86_64/",
+                        "openssh-7.7p1-1.fc29.x86_64.rpm",
```

Comment:
Exercise for you to do:

download the file at https://kojipkgs.fedoraproject.org/packages/openssh/7.7p1/1.fc29/x86_64/openssh-7.7p1-1.fc29.x86_64.rpm

I'm going to assume it's in a folder called ~/Downloads now.  We're going to make a temporary directory and extract it in there so we can take a look at it:

```
mkdir ~/tmp
cd ~/tmp
rpm2cpio ~/Downloads/openssh-7.7p1-1.fc29.x86_64.rpm | cpio -idmv
```

That should spit out a list of filenames as it extracts them.  

Take a look at the files.  There's only one thing in a bin directory that looks like we can use it, and that's ssh-keygen, so let's look at the strings in there:

```
strings usr/bin/ssh-keygen | grep -i openssh
```

hm, that's not good.  I don't see anything in there that looks like 7.7p1.  Let's see what happens if I search for 7.7...

```
strings usr/bin/ssh-keygen | grep -i 7.7
```

Well, that gets a result.  Will the existing tester detect it?  Can you use it?

From here, you can either add a regex to the checker for this new string, or you can decide that maybe this another test would be easier.  

If you want a new test: We know from looking at the binaries on my system that /usr/sbin/sshd and /usr/bin/ssh have strings of the type we're looking for.  Can you find the package that contains one or both of those files and use that as your test instead?